src/index.html

```diff
@@ -7,12 +7,56 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__image"></div>
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+      <p class="card__text">Product code: 195434</p>
+      <div class="card__item card__item--stars">
+        <ul class="stars">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+        <p class="card__item card__item--text">Reviews: 5</p>
```

Comment:
You don't need made this element modifier of card__item.
Modifier need only for change something not global ( font, color, weight etc.) 
It this case it`s can be element of card block. You have card block. Then you have card__image, card__stars, card__text, card__reviews ( or something like that). Also, about price and others element. They can be on one level. But also you can use modifier if styles of this element has small difference.